src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,673 @@
+:bug-report/inspector-page-desc-clipboard "Aqui estão os dados lidos da área de transferência."
+:bug-report/inspector-page-desc-copy "Se for seguro compartilhar, clique no botão de cópia."
+:bug-report/inspector-page-desc-create-issue "Agora você pode relatar o resultado colado na sua área de transferência. Cole o resultado na seção 'Contexto Adicional' e indique de onde você copiou o conteúdo original. Obrigado!"
+:help/title-usage "Uso"
+:help/title-community "Comunidade"
+:help/title-development "Desenvolvimento"
+:help/title-about "Sobre"
+:help/title-terms "Termos"
+:help/start "Começando"
+:help/about "Sobre o Logseq"
+:help/roadmap "Roteiro"
+:help/bug "Relatório de bug"
+:help/feature "Solicitação de recurso"
+:help/changelog "Registro de alterações"
+:help/blog "Blog do Logseq"
+:help/docs "Documentação"
+:help/privacy "Política de privacidade"
+:help/terms "Termos"
+:help/forum-community "Comunidade do Fórum"
+:help/awesome-logseq "Incrível Logseq"
+:help/shortcuts "Atalhos de teclado"
  :help/shortcuts-triggers "Gatilhos"
  :help/shortcut "Atalho"
- :help/slash-autocomplete "Auto-completar com a barra"
- :help/block-content-autocomplete "Auto-completar conteúdo de bloco (Src, Quote, Query, etc)"
- :help/reference-autocomplete "Auto-completar referência de páginas"
+ :help/slash-autocomplete "Barra de preenchimento automático"
+ :help/block-content-autocomplete "Bloquear preenchimento automático de conteúdo"
+ :help/reference-autocomplete "Preenchimento automático de referência de página"
  :help/block-reference "Referência de bloco"
+ :help/open-link-in-sidebar "Abrir link na barra lateral"
  :more "Mais"
- :search/result-for "Resultado da pesquisa para "
- :help/context-menu "Menu contextual"
+ :search/result-for "Resultado da pesquisa por"
+ :search/items "itens"
+ :search/page-names "Pesquisar nomes de páginas"
+ :search/recent "Pesquisa recente:"
+ :search/blocks-in-page "Blocos de pesquisa na página:"
+ :search/command-palette-tip-1 "Dica: "
+ :search/command-palette-tip-2 " para abrir a paleta de comandos"
+ :search/cache-outdated "O cache está desatualizado. Clique no botão 'Reindexar' no menu suspenso do gráfico."
+ :search-item/page "Página"
+ :search-item/file "Arquivo"
+ :search-item/block "Bloco"
+ :search-item/no-result "Nenhum resultado correspondente"
+ :help/context-menu "Bloquear menu de contexto"
  :help/markdown-syntax "Sintaxe de Markdown"
- :help/org-mode-syntax "Sintaxe de modo Org"
+ :help/org-mode-syntax "Sintaxe de Org mode"
  :bold "Negrito"
  :italics "Itálico"
- :highlight "Realçado"
- :strikethrough "Rasurado"
+ :highlight "Destaque"
+ :strikethrough "Tachado"
  :code "Código"
+ :untitled "Sem título"
  :right-side-bar/help "Ajuda"
- :right-side-bar/switch-theme "Temas"
+ :right-side-bar/switch-theme "Modos temáticos"
  :right-side-bar/contents "Conteúdo"
- :right-side-bar/page-graph "Grafo da página"
- :right-side-bar/block-ref "Referência de bloco"
+ :right-side-bar/page-graph "Página de grafo"
```

Comment:
Here also. I think that in Portuguese it sounds better "Grafo da página" itself. What is your opinion? Can we change it back?

Reply:
Can you send me a screenshot of where this option is in the running program? I'm Brazilian, Portuguese is my native language, if you could send me a screenshot of the entire screen showing this option I would like to describe it better, I translated it just looking at the translation file, it may change with the context but just looking at the file translation this is more natural for me, if possible, try to describe to me better what it would be in English, maybe it will help.

Reply:
Sure! Here is the screenshot. You can see the phrase "Grafo da página" in the top right corner.

![page-view](https://github.com/logseq/logseq/assets/47436525/6c162af4-d16c-46e1-925b-9b32b4b61709)

Reply:
I was able to understand, in this context it really makes more sense to use "Grafo da página" because it is about the page links, I have already made an adjustment.